src/sentry/api/event_search.py

```diff
@@ -860,6 +860,10 @@ def get_reference_event_conditions(snuba_args, event_slug):
             value = tags.get(match.group(1), None)
         else:
             value = event_data.get(field, None)
+            # If the value is a sequence use the first element as snuba
+            # doesn't support `=` or `IN` operations on fields like exception_frames.filename
+            if isinstance(value, (list, set)) and len(value):
```

Comment:
Would https://clickhouse.yandex/docs/en/query_language/functions/array_functions/#hasall help here at all? It's not perfect since it treats it like a subset, so I think the comparison is unordered, but might still work ok?

Reply:
That might work. Snuba doesn't currently support `hasall`. I'd like to see if we get any feedback from customers on whether or not they want to match more of the trace before building that.